### PR TITLE
Accessibility checker documentation

### PR DESCRIPTION
**Pull request by albinazs**
Accessibility checker documentation for the 4.2 release version.

### CODE REVIEW

File: docs/advanced_topics/accessibility_considerations.md [hunk 1]
```diff
@@ -126,6 +126,18 @@ Make sure to test your forms’ implementation with assistive technologies, and
 
 Here are things you can do to help authors create accessible content.
 
+### built-in accessibility checker
```
Comment (gasman) [inline docs/advanced_topics/accessibility_considerations.md:129]: The built-in accessibility checker isn't really a "thing you can do", so I'd suggest either re-wording the sentence above, or moving this to its own second-level section.

Also, "Built-in" should be capitalised:
```suggestion
### Built-in accessibility checker
```

File: docs/advanced_topics/accessibility_considerations.md [hunk 2]
```diff
@@ -126,6 +126,18 @@ Make sure to test your forms’ implementation with assistive technologies, and
...
+
+Wagtail includes an accessibility checker built into the [`wagtailuserbar`](wagtailuserbar_tag). The checker can help authors create more accessible websites in compliance with the Web Content Accessibility Guidelines (WCAG) 2.0 and the Authoring Tool Accessibility Guidelines (ATAG) 2.0.
```
Comment (gasman) [inline docs/advanced_topics/accessibility_considerations.md:131]: We're talking about the user bar as a user-facing feature here (not just the template tag), so we should refer to it in plain English rather than formatting it as code:

```suggestion
Wagtail includes an accessibility checker built into the [user bar](wagtailuserbar_tag). The checker can help authors create more accessible websites in compliance with the Web Content Accessibility Guidelines (WCAG) 2.0 and the Authoring Tool Accessibility Guidelines (ATAG) 2.0.
```

**Review by gasman — CHANGES_REQUESTED** [review]

File: docs/advanced_topics/accessibility_considerations.md (hunk 1 above)
Comment (albinazs) [reply docs/advanced_topics/accessibility_considerations.md:129]: Like "The process of creating accessible content can be enhanced with the use of built-in tools and additional resources."?

Comment (gasman) [reply docs/advanced_topics/accessibility_considerations.md:129]: Sounds good! But maybe a bit less wordy - "A number of built-in tools and additional resources are available to help create accessible content."

**Review by gasman — APPROVED** [review]
Looks good to me - thanks @albinazs!
